docs/src/pages/docs/Creating Charts and Dashboards/exploring-data.mdx

```diff
@@ -23,9 +23,12 @@ following information about each flight is given:
 
 ### Enabling Data Upload Functionality
 
-You may need to enable the functionality to upload a CSV or Excel file to your database. The following section
+You may need to enable the functionality to upload a CSV, Excel file, Columnar file to your database. The following section
```

Comment:
nit: I would change the comma to an "or", but I'm mostly indifferent with "Columnar" vs "Parquet" (for now we only support Parquet, but in the future we may add support for other columnar formats).
```suggestion
You may need to enable the functionality to upload a CSV, Excel file or Parquet file to your database. The following section
```

Reply:
@villebro it has been updated.